sympy/utilities/lambdify.py

```diff
@@ -50,9 +50,9 @@
     "E": "e",
     "I": "j",
     "ln": "log",
-    #"lowergamma":"lower_gamma",
+    "lowergamma":"gammainc",
```

Comment:
@jksuom i have fixed the value , now both uppergamma and lowergamma are returning different values